lib/common/ipc_common.c

```diff
@@ -16,83 +16,21 @@
 #include <crm/common/xml.h>
 #include "crmcommon_private.h"
 
-#define MIN_MSG_SIZE    12336    // sizeof(struct qb_ipc_connection_response)
```

Comment:
It's odd that this was a hard-coded value in the first place, since `sizeof` is a compile-time operator...